content/en/core/releases/4.2.0.md

```diff
@@ -0,0 +1,153 @@
+- [#7947](https://github.com/medic/cht-core/issues/7947): Improve wording of the warning when upgrading.
+- [#7964](https://github.com/medic/cht-core/issues/7964): Discuss whether to resume view indexing if installation was interrupted
+- [#8009](https://github.com/medic/cht-core/issues/8009): Display version and build number separately in the admin upgrade page
+- [#8093](https://github.com/medic/cht-core/issues/8093): Prevent data loss through re-sync when restoring data from backup
+- [#8136](https://github.com/medic/cht-core/issues/8136): Add timestamp information in feedback document log events
+- [#8210](https://github.com/medic/cht-core/issues/8210): Change extension libs API so libs are modules not functions
+- [#6281](https://github.com/medic/cht-core/issues/6281): Allow app builders to specify disabled languages in config
+- [#8134](https://github.com/medic/cht-core/issues/8134): Refactor initial replication to not use PouchDb replicate
+
+### Security fixes
+
+None.
+
+### Performance improvements
+
+None.
+
+### Bug fixes
+
+- [#7425](https://github.com/medic/cht-core/issues/7425): Schedules don't have a default value for `start_from`
+- [#7602](https://github.com/medic/cht-core/issues/7602): Race condition in db-object-widget
+- [#7603](https://github.com/medic/cht-core/issues/7603): Race condition in unloading contact content
+- [#7642](https://github.com/medic/cht-core/issues/7642): Successful users-info request reported as errored
+- [#7892](https://github.com/medic/cht-core/issues/7892): Enketo selected option highlight color changes incorrectly on drag
+- [#7910](https://github.com/medic/cht-core/issues/7910): Confirmation prompt for "Cancel" button in forms
+- [#7925](https://github.com/medic/cht-core/issues/7925): Issue in Bikram Sambat Date Picker
+- [#7949](https://github.com/medic/cht-core/issues/7949): Nginx Environment Variables not passed along
+- [#7953](https://github.com/medic/cht-core/issues/7953): 4.x: Version deployed still visible in App Management release list
+- [#7957](https://github.com/medic/cht-core/issues/7957): Sentinel erroneously reporting target for outbound push has expired TLS cert
+- [#7976](https://github.com/medic/cht-core/issues/7976): Unhandled rejection when favicon not found
+- [#7986](https://github.com/medic/cht-core/issues/7986): Container names issue for clustered couchdb setup when running across different nodes
+- [#8040](https://github.com/medic/cht-core/issues/8040): API Crashes on malformed translation docs
+- [#8154](https://github.com/medic/cht-core/issues/8154): Race condition during sentinel startup
+- [#8144](https://github.com/medic/cht-core/issues/8144): Secondary risk factors are not returned if primary risk factors are present in getRiskFactorsFromPregnancy
+- [#8130](https://github.com/medic/cht-core/issues/8130): Haproxy stops routing during stress test
+- [#225](https://github.com/medic/cht-conf/issues/225): Ability to run Tasks.js and Targets.js code without Nools
```

Comment:
This is related with this [commit](https://github.com/medic/cht-core/commit/74e6bfdcf1678db408ddf94961ec3a4dce60ae97), @garethbowen Is it okay to put it here?

Reply:
Yes!